cfgov/ask_cfpb/models/pages.py

```diff
@@ -113,51 +103,23 @@ def get_ask_breadcrumbs(
             'title': '', 'href': '/consumer-tools/{}/',
         }],
     }
-    CATEGORY_CRUMBS = {
-        'es': {
-            'title': '',
-            'href': '/es/obtener-respuestas/categoria-{}/',
-        },
-        'en': {
-            'title': '',
-            'href': '/ask-cfpb/category-{}/',
-        }
-    }
 
-    if portal_topic and flag_enabled('ASK_CATEGORIES_OFF', request=request):
+    if portal_topic:
+        page = get_portal_or_portal_search_page(
+            portal_topic=portal_topic, language=language)
         portal_title = portal_topic.title(language=language)
         portal_slug = slugify(portal_title)
         crumbs = PORTAL_CRUMBS[language]
-        portal_page = portal_topic.portal_pages.filter(
-            language=language, live=True).first()
-        if portal_page:
-            title = portal_page.title
-            slug = portal_slug
-            crumbs[0].update({
-                'title': title,
-                'href': crumbs[0]['href'].format(slug)
-            })
-        else:
-            portal_search_page = portal_topic.portal_search_pages.get(
-                language=language)
-            title = portal_search_page.title
+        title = page.title
+        href = crumbs[0]['href'].format(portal_slug)
```

Comment:
I think this is an existing issue, but it looks like generating the href with a slugified version of the portal topic title doesn't work in all cases -- the slug for the 'Fraud and scams' portal page is `fraud`, not `fraud-and-scams`, so the breadcrumb link on a [fraud and scams](http://localhost:8000/ask-cfpb/how-do-i-dispute-a-charge-on-my-credit-card-bill-en-61/) answer page 404s.

Reply:
Fixed in b39c9d6c34306a77181591e0afc1e6d400e57140, thanks for catching this!